Cargo.toml

```diff
@@ -316,6 +316,7 @@ same-file = "1.0.6"
 self_cell = "1.0.1"
 selinux = "0.4"
 signal-hook = "0.3.17"
+sysinfo = "0.29.10"
```

Comment:
please fix this error

Reply:
@tertsdiepraam i am wondering if we don't think already with a crate providing the feature, wdyt ? :)